### PR TITLE
search-1

### PR DESCRIPTION
**Pull request by vkariagin**
search-1 - added init search files

### CODE REVIEW

File: search.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Some search stuff here.
```
Comment (vkariagin) [inline search.txt:1]: you can added comments here

Comment (vkariagin) [reply search.txt:1]: @iekulyk right?

Comment (iekulyk) [reply search.txt:1]: LGTM

**Review by iekulyk — APPROVED** [review]
LGTM

**Pull request merged**